### ISSUE TITLE
"Register your team for Hackathon" box 

### ISSUE THREAD

**Issue by srajankumar** [so-sc/Sankalp#5]
Describe Problem statement must be optional for company specific.

![image](https://github.com/so-sc/Sankalp/assets/103196610/0668a929-54a4-4101-9955-f07a3ed52df0)

**Comment by dev-shetty** [comment]
Yeah, it is a feature! So we can get to know what they are expecting, in the note I have mentioned to put opinions

(Totally not because there is too much headache altering the schema and all now and I don't have time)

**Comment by AkkilMG** [comment]
@dev-shetty your getting lazy. Try fixing it nearly more than 5-6 people have already reported the issue.

**Comment by dev-shetty** [comment]
Not lazy man, I am not getting time I had to get thing out that's why it was not priority

**Comment by AkkilMG** [comment]
@dev-shetty okay bro, we need to fix it. People are getting issue.
